frontend/src/util/util.ts

```diff
@@ -0,0 +1,62 @@
+import { PresetColorType } from "antd/es/_util/colors";
+import React from "react";
+import { Requisition, RequisitionStatus } from "../types/Requisition";
+
+export const StatusToColor = (status: RequisitionStatus): PresetColorType => {
+  switch (status) {
+    case "DRAFT": return "magenta";
+    case "SUBMITTED": return "gold";
+    case "PENDING_CHANGES": return "orange";
+    case "READY_TO_ORDER": return "green";
+    case "ORDERED": return "blue";
+    case "RECEIVED": return "blue";
+    case "CANCELLED": return "red";
+    default: return "red";
+  }
+};
+
+export const StatusToString = (status: RequisitionStatus) => {
+  switch (status) {
+    case "DRAFT": return "Draft";
+    case "SUBMITTED": return "Submitted";
+    case "PENDING_CHANGES": return "Pending Changes";
+    case "READY_TO_ORDER": return "Ready to Order";
+    case "ORDERED": return "Ordered";
+    case "RECEIVED": return "Received";
+    case "CANCELLED": return "Cancelled";
+    default: return "Unknown";
+  }
+};
+
+export const StatusToStep = (status: RequisitionStatus) => {
+  switch (status) {
+    case "DRAFT": return 0;
+    case "SUBMITTED": return 1;
+    case "PENDING_CHANGES": return 1;
+    case "READY_TO_ORDER": return 2;
+    case "ORDERED": return 3;
+    case "RECEIVED": return 4;
+    case "CANCELLED": return 1;
+    default: return 0;
+  }
+};
+
+export const formatPrice = (num: number) => `$${num.toLocaleString("en-US", { minimumFractionDigits: 2, maximumFractionDigits: 2 })}`;
```

Comment:
`Number.toLocaleString` seems to have a currency option you can use instead of the hard-coded dollar sign:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString

Reply:
Good find! Fixed.